ccd-definition/CaseEventToFields/CareSupervision/messageJudge.json

```diff
@@ -94,8 +56,7 @@
     "DisplayContext": "READONLY",
     "PageID": "NewMessageNote",
     "ShowSummaryChangeOption": "N",
-    "PageDisplayOrder": 3,
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/message-judge/validate/mid-event"
```

Comment:
we can remove this because the validate only relates to replying to a message right?

Reply:
yeah, i can see it's used in the new event flow

Reply:
Yeah, only used in the reply flow